scripts/deploy-transport-controller.sh

```diff
@@ -13,19 +13,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Transport deployment script assumes it runs using the kubeflex hosting cluster context.
-
-
 DEFAULT_TRANSPORT_CONTROLLER_IMAGE=ghcr.io/kubestellar/ocm-transport-plugin/transport-controller:0.1.7
 
+set_context=""
```

Comment:
set_context is confusing.  seems like some Boolean flag.   tc-context ?  or even just "context"  ?

Reply:
The idea was to not mess with `--context` if not requested.

But it was buggy. So I have changed it to always give `--context` to kubectl.